raiden/network/upnpsock.py

```diff
@@ -8,6 +9,17 @@
 log = slogging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def valid_ip_v4(address):
+    try:
+        ipaddress.ip_address(address)
```

Comment:
Oh, that is correct. I'm afraid the `.is_global/private` attributes won't help though. I re-added the `0.0.0.0` address check and also renamed the function to reflect that we look for mappable addresses.